egret/model_library/unit_commitment/generation_limits.py

```diff
@@ -34,17 +35,33 @@ def enforce_generator_output_limits_rule_part_a(m, g, t):
 
 
 def _MLR_generation_limits_uptime_1(model, tightened=False):
+    if (is_var(model.MaximumPowerAvailable) or is_var(model.MaximumPowerAvailableAboveMinimum)) and\
```

Comment:
The fact that you generally can have a mix of ands and ors of participating variables probably nixes my desire for a easy-to-use wrapper around the expression-selector logic...

Reply:
If we eliminated some modeling choices that aren't used in the higher-level functions (and are not tested), we could use "ands" everywhere for the linear expressions selector (as either PowerOutput or PowerOutputAboveMinimum) would be a Var. Hence the selector logic could be put in uc_utils. Thoughts?